http-server/src/main/java/io/airlift/http/server/HttpServer.java

```diff
@@ -143,6 +145,16 @@ public HttpServer(
         threadPool.setIdleTimeout(toIntExact(config.getThreadMaxIdleTime().toMillis()));
         threadPool.setName("http-worker");
         threadPool.setDetailedDump(true);
+        if (enableVirtualThreads) {
+            Executor executor = getNamedVirtualThreadsExecutor("http-worker#v");
+            if (executor == null) {
+                log.info("Virtual threads support is not available");
```

Comment:
Should this be an error? Since we do the version check, this shouldn't be possible.